src/main/kotlin/com/cognifide/gradle/aem/bundle/BundleCollector.kt

```diff
@@ -27,17 +26,10 @@ class BundleCollector(val project: Project) {
     val allJars: Collection<File>
         get() {
             val jars = TreeSet<File>()
-            archiveConfig?.apply { jars += allArtifacts.files.files }
```

Comment:
bundles that comes from that stage are not exisitng yet... they will be built in execution phase, now  this BundleCollector is being called in configuration phase to be able to detect JAR changes before zipping CRX package